#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_mythgallery_ubuntu_directories.dpatch by laga <>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change some paths to match Debian/Ubuntu better

@DPATCH@

Index: mythgallery/gallerysettings.cpp
===================================================================
--- a/mythgallery/mythgallery/gallerysettings.cpp	(revision 13505)
+++ b/mythgallery/mythgallery/gallerysettings.cpp	(working copy)
@@ -18,7 +18,7 @@
 #ifdef Q_WS_MACX
     gc->setValue(QDir::homeDirPath() + "/Pictures");
 #else
-    gc->setValue("/var/lib/pictures");
+    gc->setValue("/var/lib/mythtv/pictures");
 #endif
     gc->setHelpText(QObject::tr("This directory must exist and "
                        "MythGallery needs to have read permission."));
@@ -71,7 +71,7 @@
 {
     HostLineEdit *gc = new HostLineEdit("GalleryImportDirs");
     gc->setLabel(QObject::tr("Paths to import images from"));
-    gc->setValue("/mnt/cdrom:/mnt/camera");
+    gc->setValue("/media/cdrom:/media/usbdisk");
     gc->setHelpText(QObject::tr("This is a colon separated list of paths. "
                     "If the path in the list is a directory, its contents will "
                     "be copied. If it is an executable, it will be run."));
